metis-enrichment/metis-enrichment-rest/src/test/java/eu/europeana/enrichment/rest/EnrichmentControllerTest.java

```diff
@@ -112,52 +111,52 @@ public void getByUri_JSON_throwsException() throws Exception {
   @Test
   public void getByUri_XML() throws Exception {
     EntityWrapper wrapper = new EntityWrapper();
-    String uri = "http://www.fennek-it.nl";
+    String uri = "http://www.example.com";
     when(enrichmerMock.getByUri(uri)).thenReturn(wrapper);
     Agent agent = getAgent(uri);
     Map<String, String> namespaceMap = getNamespaceMap();
     when(converterMock.convert(wrapper)).thenReturn(agent);
     enrichmentControllerMock.perform(get("/getByUri")
-        .param("uri", "http://www.fennek-it.nl")
+        .param("uri", "http://www.example.com")
         .accept(MediaType.APPLICATION_XML_VALUE))
         .andExpect(status().is(200))
-        .andExpect(xpath("edm:Agent/@rdf:about", namespaceMap).string("http://www.fennek-it.nl"))
+        .andExpect(xpath("edm:Agent/@rdf:about", namespaceMap).string("http://www.example.com"))
         .andExpect(xpath("edm:Agent/skos:altLabel[@xml:lang='en']", namespaceMap).string("labelEn"))
         .andExpect(
             xpath("edm:Agent/skos:altLabel[@xml:lang='nl']", namespaceMap).string("labelNl"));
   }
 
-  @Test
-  public void enrich_XML() throws Exception {
-    String body =
-          "{\n"
-        + "  \"inputValueList\": [\n"
-        + "    {\n"
-        + "      \"value\": \"Music\",\n"
-        + "      \"vocabularies\": [\n"
-        + "        \"CONCEPT\"\n"
-        + "      ]\n"
-        + "    }\n"
-        + "  ]\n"
-        + "}";
-
-    Map<String, String> namespaceMap = getNamespaceMap();
-
-    EnrichmentResultList enrichmentResultList = new EnrichmentResultList();
-    enrichmentResultList.getResult().add(getAgent("http://www.fennek-it.nl"));
-
-    when(converterMock.convert(anyList())).thenReturn(enrichmentResultList);
-
-    enrichmentControllerMock.perform(post("/enrich")
-        .content(body)
-        .accept(MediaType.APPLICATION_XML_VALUE)
-        .contentType(MediaType.APPLICATION_JSON_UTF8_VALUE))
-        .andExpect(status().is(200))
-        .andExpect(xpath("metis:results/edm:Agent/@rdf:about", namespaceMap).string("http://www.fennek-it.nl"))
-        .andExpect(xpath("metis:results/edm:Agent/skos:altLabel[@xml:lang='en']", namespaceMap).string("labelEn"))
-        .andExpect(xpath("metis:results/edm:Agent/skos:altLabel[@xml:lang='nl']", namespaceMap).string("labelNl"))
-        .andExpect(xpath("metis:results/edm:Agent/rdaGr2:dateOfBirth[@xml:lang='en']", namespaceMap).string("10-10-10"));
-  }
+//  @Test
```

Comment:
Instead of commenting this out, wouldn't it be better to remove the @test annotation, or add a @ignore to it? Or, if we don't need it, why not completely remove it?

Reply:
Actually it was a mistake from me by not removing the comment lines. Fixed the test now.